cli/cmd/cluster/certificate-rotator.go

```diff
@@ -141,7 +145,6 @@ func (cr *certificateRotator) restartDaemonSetAndWaitToConverge(ns, name string)
 		return fmt.Errorf("restarting: %w", err)
 	}
 
-	// TODO: make sure this is the right value.
```

Comment:
Did we check if those values are right? Sleep is rather a hack here, but I couldn't find a better way to detect if the deployment has started rolling the updates already.

Reply:
We can look at the generation number?

Reply:
>We can look at the generation number?

Perhaps. I couldn't quite figure it out the last time and I didn't want to spend too much time on this :smile: Happy to test any approach you figure out :)

Reply:
Okay! good to know the context will try some things out I thought a lot more research time was put into this :D